### ISSUE TITLE
[Feature request] Automatically detect type of imported token

### ISSUE THREAD

**Issue by mikhirev** [helloworld1/FreeOTPPlus#279]
It can be confusing for non-technical users to select what they need to import, JSON or URI. It would be much better if the app detected the type of imported content itself.

**Comment by helloworld1** [comment]
Agreed. It is relatively easy technically to auto detect. But, it is better to tell user what format can be imported to prevent user importing incompatible files
